stuf_zds_payments/registration_variables.py

```diff
@@ -57,3 +57,15 @@ def get_initial_value(self, submission: Submission | None = None):
         if submission is None:
             return None
         return submission.payments.get_completed_public_order_ids()
+
+
+@register("provider_payment_ids")
+class ProviderPaymentIds(BaseStaticVariable):
+    name = _("Provider payment IDs")
+    data_type = FormVariableDataTypes.array
+
+    def get_initial_value(self, submission: Submission | None = None):
+        if submission is None:
+            return None
+
+        return list(submission.payments.values_list("provider_payment_id", flat=True))
```

Comment:
I think this now includes all the payments, also payments that have failed. We should only include the completed (and there's a second status) payments.

Reply:
ah that's right, I added the same filter used by `submission.payments.get_completed_public_order_ids`